client/landing/stepper/declarative-flow/internals/steps-repository/readymade-template-generate-content/style.scss

```diff
@@ -0,0 +1,118 @@
+@import "../style";
+
+$font-family: "SF Pro Text", $sans;
```

Comment:
This font-family var looks unused as well

Reply:
Removed.